BugDoom - web app/src/main/java/com/infoshareacademy/pl/model/Track.java

```diff
@@ -89,37 +98,20 @@ public String toString() {
                 ", length=" + length +
                 ", difficulty='" + difficulty + '\'' +
                 ", terrain='" + terrain + '\'' +
+                ", eventId=" + eventId +
                 '}';
     }
 
     @Override
     public boolean equals(Object o) {
         if (this == o) return true;
         if (o == null || getClass() != o.getClass()) return false;
-
         Track track = (Track) o;
-
-        if (trackId != track.trackId) return false;
-        if (length != track.length) return false;
-        if (!Objects.equals(competitionName, track.competitionName))
-            return false;
-        if (!Objects.equals(start, track.start)) return false;
-        if (!Objects.equals(finish, track.finish)) return false;
-        if (!Objects.equals(checkpoints, track.checkpoints)) return false;
-        if (!Objects.equals(difficulty, track.difficulty)) return false;
-        return Objects.equals(terrain, track.terrain);
+        return trackId == track.trackId && length == track.length && eventId == track.eventId && Objects.equals(competitionName, track.competitionName) && Objects.equals(start, track.start) && Objects.equals(finish, track.finish) && Objects.equals(checkpoints, track.checkpoints) && Objects.equals(difficulty, track.difficulty) && Objects.equals(terrain, track.terrain);
```

Comment:
hmmm, może jest sposób, żeby to zapisać czytelniej

Reply:
zmieniłem wzór do generowania tych dwóch metod, żeby było bardziej czytelnie (czyli użyłem tego, który był wcześniej)